corehq/apps/accounting/static/accounting/js/credits-tab.js

```diff
@@ -1,13 +1,16 @@
-var CreditFormModel = function () {
-    var self = this;
-    self.rateType = ko.observable("");
-    self.showProduct = ko.computed(function() {
-        return self.rateType() == 'Product';
-    }, self);
-    self.showFeature = ko.computed(function() {
-        return self.rateType() == 'Feature';
-    }, self);
-};
+var $form = $('#credit-form');
+if ($form.length) {
+    var CreditFormModel = function () {
```

Comment:
good point. just did a grep, and this is the only place it's used. do you think it would be an improvement to wrap this file in a closure so that's more obvious?